core/web/src/main/java/org/opennms/core/web/HttpClientWrapper.java

```diff
@@ -388,7 +415,7 @@ public CloseableHttpClient getClient() {
                 requestConfigBuilder.setConnectTimeout(m_connectionTimeout);
             }
             if (m_retries != null) {
-                httpClientBuilder.setRetryHandler(new DefaultHttpRequestRetryHandler(m_retries, false));
+                httpClientBuilder.setRetryHandler(new HttpRequestRetryOnExceptionHandler(m_retries, true));
```

Comment:
Here's a list of all of the stuff using HttpClientWrapper:
```
JUnitHttpServerTest
HttpClientWrapper
UsageStatisticsReporter
NominatimGeocoderService
NominatimGeocoder
HypericAckProcessor
HttpNorthbounder
HttpCollector
NCSNorthbounder
HttpNotificationStrategy
HttpPostMonitor
PageSequenceMonitor
WebMonitor
WebClient
RequestTracker
HttpUrlConnection
HttpUrlConnectionIT
```
For a lot of these, requestSentRetryEnabled would be fine since it's just idempotent read operations but I guess it could cause problems with alarm forwarding and notifications in some rare cases. I'll turn it back off.